react/src/lib/components/DeckGLMap/layers/map/terrainmap.fs.glsl.ts

```diff
@@ -99,11 +108,11 @@ void main(void) {
    }
 
 
-   bool is_contours = contourReferencePoint != -1.0 && contourInterval != -1.0;
+   bool is_contours = true; //contourReferencePoint != -1.0 && contourInterval != -1.0;
```

Comment:
Can't disable contours with this change.

Reply:
I completely forgot to comment this in again!